src/api/schemas/business-schema.ts

```diff
@@ -0,0 +1,121 @@
+import { object, string } from 'zod';
+
+const employeeSchema = object({
+    firebase_id: string({
+        required_error: 'Firebase ID is required',
+        invalid_type_error: 'Firebase ID should be of type string'
+    }),
+    firstName: string({
+        required_error: 'First Name is required',
+        invalid_type_error: 'First Name should be of type string'
+    }).min(2, 'First Name too short!'),
+    lastName: string({
+        required_error: 'Last Name is required',
+        invalid_type_error: 'Last Name should be of type string'
+    }).min(2, 'Last Name too short!'),
+    email: string({
+        required_error: 'Email is required',
+        invalid_type_error: 'Email should be of type string'
+    }).email('Invalid email'),
+    role: string({
+        required_error: 'Role is required',
+        invalid_type_error: 'Role should be of type string'
+    })
+});
+
+const addressSchema = object({
+    // Required
+    lineOne: string({
```

Comment:
What are lineOne and lineTwo? Maybe put in comment a small description.

Reply:
it's line one and line two in normal convention for address